admin/eliotlogging.py

```diff
@@ -0,0 +1,51 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Filter Eliot logs out of Twisted test.log.
+"""
+
+import sys
+
+from flocker.testtools._base import _get_eliot_data
```

Comment:
Done both. I wish Eliot had terms that were better defined, because I don't really know what to call a JSON-encoded Eliot message/action.